web/beacon-app/src/features/onboarding/components/OnboardingStepperStep.tsx

```diff
@@ -0,0 +1,24 @@
+import { Trans } from '@lingui/macro';
+
+interface OnboardingStepperStepProps {
+  title: string;
+  description: string;
+}
+
+const OnboardingStepperStep = ({ title, description }: OnboardingStepperStepProps) => {
```

Comment:
I didn't see a Stepper component that could be imported from `flowbite` or `flowbite-react`. The code is from Flowbite's site with some minor changes to make the step indicator smaller.